server.js

```diff
@@ -1,345 +1,42 @@
 const express = require("express");
 const Sequelize = require("sequelize");
-const bodyParser = require("body-parser");
-const bcrypt = require("bcrypt-nodejs");
-var cors = require("cors");
-const _ = require("lodash");
 
-const queue = require("./messaging");
 
 // Pick and environment out of "development", "test", "production" and load configuration accordingly
 var env = "development";
 const connConfig = require("./config/connection")[env];
 
 // Setup a database connection using loaded configuration
 const sequelize = new Sequelize(
-  connConfig.database,
-  connConfig.username,
-  connConfig.password,
-  {
-    dialect: connConfig.dialect,
-    host: connConfig.host,
-    operatorsAliases: false
-  }
+    connConfig.database,
+    connConfig.username,
+    connConfig.password,
+    {
+        dialect: connConfig.dialect,
+        host: connConfig.host,
+        operatorsAliases: false
+    }
 );
 
-const { UserController } = sequelize.import("./controllers/user");
-const { FollowController } = sequelize.import("./controllers/follow");
-
 const app = express();
-app.use(bodyParser.json());
-app.use(cors());
-
-// user create, get, update, delete routes
-app.post("/user", (req, res) => {
-  params = _.pick(req.body, [
-    "userName",
-    "password",
-    "firstName",
-    "lastName",
-    "bio",
-    "dob",
-    "location"
-  ]);
-  if (!params.userName)
-    return res
-      .status(400)
-      .send({ error: { name: "userName is not specified" } });
-  if (!params.password)
-    return res
-      .status(400)
-      .send({ error: { name: "password is not specified" } });
-  UserController.create(req.body)
-    .then(user => {
-      delete user.dataValues.password;
-      res
-        .set("Authorization", "Bearer fake-jwt-token")
-        .status(201)
-        .send({ user });
-    })
-    .catch(error => {
-      if (error.name === "SequelizeUniqueConstraintError")
-        return res.status(409).send({ error });
-      res.status(500).send({ error });
-    });
-});
-
-app.post("/user/search", (req, res) => {
-  var searchQuery = req.body.searchQuery;
-  var userId = req.body.userId;
-  console.log("Search endpoint hit");
-  if (!searchQuery)
-    return res
-      .status(400)
-      .send({ error: { name: "searchQuery is not specified" } });
-  if (!userId)
-    return res.status(400).send({ error: { name: "userId is not specified" } });
 
-  UserController.find({
-    userName: { [Sequelize.Op.like]: "%" + searchQuery + "%" }
-  })
-    .then(searchResult => {
-      FollowController.find({ follower: userId }).then(users => {
-        followingList = Object.values(users.map(v => v.dataValues.following));
-        console.log(typeof followingList);
-
-        searchResult.forEach(result => {
-          delete result.dataValues.password;
-          result.dataValues.following = followingList.includes(
-            result.dataValues.userId
-          );
+sequelize.sync().then(
```

Comment:
Put this block just above the module.exports section below (after all the routes have been setup)

Reply:
Made changes in [81ff7b5](/airavata-courses/ABC/pull/44/commits/81ff7b5be90c80017bc7cbbbcb3e1de75c8e9574)